src/form-builder/AddressField.tsx

```diff
@@ -3,7 +3,9 @@ import { useField, FieldHookConfig, useFormikContext } from 'formik';
 
 import { chainValidations, required } from '../utils/validation';
 import { AddressProps, Address } from './types';
-import { TextField, CheckboxField, SelectField } from './index';
+import TextField from './TextField';
+import CheckboxField from './CheckboxField';
+import SelectField from './SelectField';
```

Comment:
Cool that this fixes that warning, but I don't understand why it does lol. Can you explain it?

Reply:
Yeah so since it was importing all three `from './index` the build output was `warning` us of a circular dependency like this: 
![image](https://user-images.githubusercontent.com/22844722/172422571-8ff11d4c-c741-4fcc-8f64-c369f1b6c89b.png)